### ISSUE TITLE
Can not access unsecure_export_savedata.tox

### ISSUE THREAD

**Issue by audricd** [zoff99/ToxAndroidRefImpl#564]
**Describe the bug**
Trying to synch with a desktop client, I want my savedata.tox from Trifa

**To Reproduce**
1. Go to "Maintenance"
2. Click on "Export tox savedata *unsecure*"
3. Try to access (with FX Explorer) "/storage/emulated/0/Android/data"
4. See error "Access denied"

**Expected behavior**
I should be able to access the directory in which the savedata is exported, so I can share it to my desktop

**Your system information:**
 - TRIfA Version: [1.0.240]
 - App from Google-Play / F-Droid / Github-Releases / other: Google Play
 - Android OS Version: Android 14
 - Device: Oppo 98

**Comment by zoff99** [comment]
yeah this is not optimal. let me have a look later to check for a solution ...

**Comment by audricd** [comment]
thanks! let me know if I can provide you with anything further.

**Comment by zoff99** [comment]
if you have a usb cable and access to "adb". can you try:

```
adb shell
cd /sdcard/Android/data/com.zoffcc.applications.trifa/files/vfs_export
ls -al unsecure_export_savedata.tox
```

you can pull the file to your local system like this:

```
adb pull /sdcard/Android/data/com.zoffcc.applications.trifa/files/vfs_export/unsecure_export_savedata.tox ./
```

this is just a workaround for now.

**Comment by audricd** [comment]
thanks! that worked perfectly

**Comment by zoff99** [comment]
sadly google changed android again and now most fileexplorers are not allowed to access this directory anymore.
Meh 🙁

**Comment by zoff99** [comment]
just a quick question, you filled out that you have 1.0.240 from google play? is that a typo? the last version on google play is 1.0.231
